waterlog/src/reducers/index.js

```diff
@@ -2,15 +2,21 @@ import { combineReducers } from 'redux';
 import SegmentLeaksReducer from './SegmentLeaksReducer';
 import SegmentLeaksByIdReducer from './SegmentLeaksByIdReducer';
 import LeakLitresReducer from './LeakLitresReducer';
+import SegmentLeaksHistoryReducer from './SegmentLeaksHistoryReducer';
 import LeaksResolvedReducer from './LeaksResolvedReducer';
 import SegmentEventsReducer from './SegmentEventsReducer';
 
 export default combineReducers({
 	leaks: SegmentLeaksReducer,
 	leak: SegmentLeaksByIdReducer,
 	litres: LeakLitresReducer,
+<<<<<<< HEAD
```

Comment:
You have merge conflicts. Attend them before pushing your code.

Reply:
Done